card/db/migrate_core_cards/20150729143608_import_new_pointer_option.rb

```diff
@@ -0,0 +1,9 @@
+# -*- encoding : utf-8 -*-
+
+class ImportNewPointerOption < Card::CoreMigration
+  def up
+    options_card = Card.fetch "*input+*right+*options"
+    options_card<<"single_autocomplete"
```

Comment:
I guess what you use here as name will show up in the editor as option so if "single autocomplete" is the right name for this then better without underscore.